### DISCUSSION TITLE
v0.3.4

### DISCUSSION THREAD

**Discussion by Ljzd-PRO** [Ljzd-PRO/KToolBox#35] [Announcements]
### Changes

- Added the `JobConfiguration.sequential_filename` configuration option for renaming attachments in numerical order #33
  - Set `KTOOLBOX_JOB__SEQUENTIAL_FILENAME=True` to enable
  - 📖More information: [Configuration-Reference-JobConfiguration](https://ktoolbox.readthedocs.io/latest/configuration/reference/#ktoolbox.configuration.JobConfiguration.sequential_filename)

**Full Changelog**: https://github.com/Ljzd-PRO/KToolBox/compare/v0.3.2...v0.3.4

<hr /><em>This discussion was created from the release <a href='https://github.com/Ljzd-PRO/KToolBox/releases/tag/v0.3.4'>v0.3.4</a>.</em>

**Comment by IsFaser** [comment]
fantastic tool. however, the unit of `tqdm` seems to be incorrect. Could you consider switching to a base-1024 system using `unit_divisor`?
棒的工具. 但是`tqdm`的单位似乎不正确, 考虑使用`unit_divisor`更换为1024进制?

**Reply by IsFaser** [reply]
此外，可以增加一个选项来下载全部历史修订版本吗（`<<_basic_url>>/<<service>>/user/<<creator_id>>/post/<<post_id>>/revisions`）？我知道这有点奇怪...所以这只是个请求，不需要完成它。

Additionally, could you add an option to download historical revisions as well (`<<_basic_url>>/<<service>>/user/<<creator_id>>/post/<<post_id>>/revisions`)? I know it's a bit odd... so this is just a request, no need to fulfill it.

**Reply by Ljzd-PRO** [reply]
好的，我试试

**Comment by Nacosia** [comment]
嗨, 我在文档中看到"--update-with="用于指定更新一个作者目录, 但是我在代码中发现是使用"update_from", 这是正确的吗. 我单独新建一个文件用于测试, 实际上并没有发生工作.

**Reply by Ljzd-PRO** [reply]
这个其实是个设计失误😂 之后应该会删除，因为后面发现即使不使用这个参数，也可以对已下载的作者目录和作品目录进行更新，已存在的文件会被跳过，剩下的就是待下载的文件了
